### PR TITLE
Merged ActivitySummary into ActivityDaily for full-featured arbitrary timeperiods (also improvements to categorization settings & Home view)

### PR DESCRIPTION
**Pull request by ErikBjare**
This PR is full of gold.

 - The functionality of ActivitySummary has been merged into ActivityDaily, giving you the ability to get an fully-featured overview of a week/month of data at a time. Including categorization, browser activity, and everything else that exists in the ActivityDaily view.
 - The periodusage visualization is now a lot snappier since it doesn't re-query already queried timeperiods.
 - The Home view is updated with more links, including the user survey I've made.
 - Added a `ignore_case` toggle to categorization rules, making it more user-friendly to write category regex rules where you don't care about case (i.e. `[Aa]ctivity[Ww]atch` or `(?i)activitywatch` just becomes `activitywatch`).

### CODE REVIEW

**Comment by ErikBjare** [comment]
Holy shit, I had no idea I was that close to merging ActivitySummary into ActivityDaily after the massive refactoring I've been doing with vuex and everything recently.

This is fucking amazing. Seriously.

File: src/util/classes.ts [hunk 1]
```diff
@@ -2,35 +2,38 @@ const _ = require('lodash');
...
+  { name: ['Work'], rule: { type: 'regex', regex: 'Google Docs' } },
   {
     name: ['Work', 'Programming'],
-    rule: { type: 'regex', pattern: 'GitHub|Stack Overflow' },
+    rule: { type: 'regex', regex: 'GitHub|Stack Overflow' },
   },
   {
     name: ['Work', 'Programming', 'ActivityWatch'],
-    rule: { type: 'regex', pattern: '[Aa]ctivity[Ww]atch|aw-' },
+    rule: { type: 'regex', regex: 'ActivityWatch|aw-', ignore_case: true },
```
Comment (johan-bjareholt) [inline src/util/classes.ts]: This ignore_case breaks the query on my machine as it's directly converted to a string and the aw-core expects "True" while this is sent as a "true".

**Comment by ErikBjare** [comment]
Yeah I had that ignore_case thing as well and I just added the lowercase
booleans as constants in aw-server. Kinda makes sense to try and be JSON
compliant? Idk.

**Comment by ErikBjare** [comment]
I want to make a release today and would like to have these changes in it. We will have to take the `True`/`False` vs `true`/`false` discussion somewhere else (but we'll probably go with `true`/`false`).

**Comment by johan-bjareholt** [comment]
> I want to make a release today and would like to have these changes in it. We will have to take the True/False vs true/false discussion somewhere else (but we'll probably go with true/false).

It shouldn't be more than a 2-line change in aw-server and aw-server-rust, it's a quick fix.

If you do fix that you can merge without a review from me.

**Pull request merged**